int32 goal
---
geometry_msgs/Point32[] list_of_odoms
---
float32 current_total